### PR TITLE
Ubuntu 22.04

### PR DESCRIPTION
**Pull request by masa213f**
Signed-off-by: Masayuki Ishii <masa213f@gmail.com>

### CODE REVIEW

File: Dockerfile.runner [hunk 1]
```diff
@@ -37,7 +37,6 @@ COPY include-bird /etc/dpkg/dpkg.cfg.d/
 
 RUN apt-get update \
     && apt-get install -y software-properties-common \
-    && add-apt-repository -y ppa:smoser/swtpm \
```
Comment (masa213f) [inline Dockerfile.runner:40]: From 22.04, we need not add this repository.

https://gist.github.com/smoser/ef279e77b068cc081d2f63c10f809db0
> swtpm is available in Jammy (Ubuntu 22.04). ([ITP #941199](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=941199), [debian](https://tracker.debian.org/pkg/swtpm), [ubuntu](https://launchpad.net/ubuntu/+source/swtpm))

Comment (masa213f) [inline Dockerfile.runner:40]: From 22.04, we need not add this repository.

https://gist.github.com/smoser/ef279e77b068cc081d2f63c10f809db0
> swtpm is available in Jammy (Ubuntu 22.04). ([ITP #941199](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=941199), [debian](https://tracker.debian.org/pkg/swtpm), [ubuntu](https://launchpad.net/ubuntu/+source/swtpm))

Comment (kmdkuk) [reply Dockerfile.runner:40]: If `add-apt-repository` is no longer needed, it is likely that the installation of `software-properties-common` is no longer needed too.

Comment (masa213f) [reply Dockerfile.runner:40]: Thank you. removed 3dfa120

**Review by kmdkuk — APPROVED** [review]
LGTM

**Pull request merged**